nucypher-core-wasm/Cargo.toml

```diff
@@ -20,14 +20,15 @@ default = ["console_error_panic_hook"]
 
 [dependencies]
 # TODO: Replace with a new release after this PR is merged and released: https://github.com/nucypher/rust-umbral/pull/122
-umbral-pre = { git = "https://github.com/piotr-roslaniec/rust-umbral.git", rev = "90a5075bb3462421fa6db79463960cd8e10a7025", features = ["bindings-wasm"] }
-ferveo = { package = "ferveo-pre-release", version = "0.1.0-alpha.7", features = ["bindings-wasm"] }
+umbral-pre = { git = "https://github.com/piotr-roslaniec/rust-umbral.git", rev = "5a279039d1e44e415f2cfac85282fc5102d41a6c", features = ["bindings-wasm"] }
+ferveo = { package = "ferveo-pre-release", git = "https://github.com/nucypher/ferveo.git", rev = "9795e2969f525fc7ee3a7721dca368287f4825c8", features = ["bindings-wasm"] }
```

Comment:
Has testing been successful, and these vcs dependencies are no longer necessary?